README.md

```diff
@@ -245,35 +166,16 @@ Feel free to check the [issues page](../../issues/).
 
 ## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
 
-If you like this project...
+Superb Thanks to microverser organization for providing an opportunity to learn web development professionally. If you like this project kindly leave a star.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank...
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- FAQ (optional) -->
-
-## ❓ FAQ (OPTIONAL) <a name="faq"></a>
-
-> Add at least 2 questions new developers would ask when they decide to use your project.
-
-- **[Question_1]**
-
-  - [Answer_1]
-
-- **[Question_2]**
-
-  - [Answer_2]
+I would like to thank the maintainers of ruby on rails, react and redux-toolkit for providing amazing open source programs
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/212623559-9ab438ee-c687-45c0-a22d-a6c962757de6.png)

- [ ] Your license file is looking like a template. Kindly add your **name**, **current year** and **app-type** in the license file to make your project licensed :+1: